docs/cvl/cvl2/changes.md

```diff
@@ -142,34 +133,52 @@ rule example {
 ```
 
 Entries in the `methods` block may use either the contract name or the instance
-variable.
+variable:
 
-```{todo}
-Error message
+```cvl
+using Example as c;
+
+methods {
+    //// both are valid:
+    function c.balanceOf(address) external returns(uint) envfree;
+    function Example.transfer(address,uint) external envfree;
+}
```

Comment:
Maybe it is worth it to mention somewhere that the semantics of both function entries are the same for now, but may diverge in the future once contract instances are no longer singleton?